ome-xml/src/main/java/ome/xml/meta/MetadataConverter.java

```diff
@@ -106,6 +106,267 @@ public static void convertMetadata(MetadataRetrieve src, MetadataStore dest) {
     convertRootAttributes(src, dest);
   }
 
+  /**
+   * Copy all metadata from the specified channel in the source MetadataRetrieve
+   * to the specified channel in the destination MetadataStore.  The specified
+   * destination Image must exist (e.g. via calling
+   * <code>setImageID(..., destImage)</code>), but the destination Channel only
+   * needs to exist if the copyID flag is <code>false<code>
+   * LightSource references are assumed to be correct and are not checked for validity.
+   *
+   * @param src the source {@link MetadataRetrieve}
+   * @param srcImage the index of the Channel's parent Image in src
+   * @param srcChannel the index of the Channel in src
+   * @param dest the destination {@link MetadataStore}
+   * @param destImage the index of the Channel's parent Image in dest
+   * @param destChannel the index of the Channel in dest
+   * @param copyID false if the Channel ID should be omitted from the copy operation
+   */
+  public static void convertChannelMetadata(MetadataRetrieve src, int srcImage, int srcChannel,
```

Comment:
All other methods of this class are called `convert<Objects>`. Should this become `convertChannels`?

Reply:
Done in abf622e.